public/samples/DataStreams/StreamsUpkeep.sol

```diff
@@ -75,28 +63,28 @@ contract StreamsUpkeep is ILogAutomation, StreamsLookupCompatibleInterface {
         address quoteAddress;
     }
 
-    event ReportVerified(BasicReport indexed report);
-    event PriceUpdate(int192 price);
+    event PriceUpdate(int192 indexed price);
```

Comment:
Since this is a demo, we could make it easier on the user to not have to switch to a blockexplorer to find the price received event.  Maybe in addition to the event,  a `s_priceUdate` storage variable that they can simple "Read" ?

Reply:
Added